Makefile

```diff
@@ -42,10 +42,13 @@ FUSES      = -U hfuse:w:0xd8:m -U lfuse:w:0xff:m
 # Tune the lines below only if you know what you are doing:
 
 AVRDUDE = avrdude  $(PROGRAMMER) -p $(DEVICE) -B 10 -F -D
-COMPILE = avr-gcc -Wall -Os -DF_CPU=$(CLOCK) -mmcu=$(DEVICE) -I. -ffunction-sections
+CFLAGS = -Wall -Os -DF_CPU=$(CLOCK) -mmcu=$(DEVICE) -I
+COMPILE = avr-gcc $(CFLAGS) -ffunction-sections
```

Comment:
I need to understand more about Makefiles before giving a more detailed response. The short response is that the `debug` mode is applied to `CFLAGS` (the `-g`) before the compile lines executes.